### PR TITLE
Change placeholder schema for organisations

### PR DESCRIPTION
**Pull request by tvararu**
Modify the placeholder schema to accommodate the branding / logo information on the organisation model. Also adds an example.

https://trello.com/c/cJGrtnc1

### CODE REVIEW

File: formats/organisation/publisher/links.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (boffbowsh) [inline formats/organisation/publisher/links.json]: This file isn't required. If it's not provided, the schema will be generated with an empty links hash.

File: formats/organisation/frontend/examples/bona_vacantia.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "phase": "live",
+  "analytics_identifier": "OT564",
+  "links": {
+    "available_translations": [
+      {
+        "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+        "title": "Bona Vacantia",
+        "base_path": "/government/organisations/bona-vacantia",
+        "description": null,
+        "api_url": "https://www-origin.integration.publishing.service.gov.uk/api/content/government/organisations/bona-vacantia",
```
Comment (boffbowsh) [inline formats/organisation/frontend/examples/bona_vacantia.json]: These URLs should be for the public site, so `www.gov.uk`

File: formats/organisation/frontend/examples/bona_vacantia.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "base_path": "/government/organisations/bona-vacantia",
+  "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+  "title": "Bona Vacantia",
+  "format": "organisation",
+  "document_type": "organisation",
```
Comment (boffbowsh) [inline formats/organisation/frontend/examples/bona_vacantia.json]: For reference, as discussed in person, this should be `placeholder_organisation`

File: formats/organisation/frontend/examples/bona_vacantia.json (hunk 2 above)
Comment (tvararu) [inline formats/organisation/frontend/examples/bona_vacantia.json]: :+1:

**Comment by tvararu** [comment]
@boffbowsh Updated!

**Comment by tijmenb** [comment]
Sorry if I jump in here, but would having a content-item with `document_type: "organisation"` and `schema_name: "placeholder"` make sense? What's the way forward with these names?

**Comment by tvararu** [comment]
@tijmenb valid concern! After discussing with @boffbowsh we've decided to not go in this direction anymore. We'll be using the `placeholder` format and just adding an `organisation.json` example.

File: formats/placeholder/frontend/examples/organisation.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+  "base_path": "/government/organisations/bona-vacantia",
+  "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+  "title": "Bona Vacantia",
+  "format": "placeholder_organisation",
+  "document_type": "organisation",
+  "schema_name": "placeholder",
```
Comment (tvararu) [inline formats/placeholder/frontend/examples/organisation.json]: @tijmenb @danielroseman: Any thoughts about using `"format": "placeholder_organisation"`?

Comment (boffbowsh) [inline formats/placeholder/frontend/examples/organisation.json]: To clarify this, I think the `format` is ok, but I don't have a huge amount of context yet on the `document_type`/`schema_name` split. Does this look ok to you @danielroseman?

Comment (tijmenb) [inline formats/placeholder/frontend/examples/organisation.json]: When you send something to publishing-api you can specify `format` OR `document_type` & `schema_name`.

In the last case, `format` will be set to `schema_name`. So the example could read:

```
+  "format": "placeholder_organisation",
+  "document_type": "organisation",
+  "schema_name": "placeholder_organisation",
```

or

```
+  "format": "placeholder",
+  "document_type": "organisation",
+  "schema_name": "placeholder",
```

**Comment by tijmenb** [comment]
@tvararu @boffbowsh 

If organisations will get extra fields ([here](https://github.com/alphagov/whitehall/blob/organisation-logos/app/presenters/publishing_api_presenters/organisation.rb#L6-L14)), shouldn't it get its own schema? I believe `"additionalProperties": true` will make `placeholder` into a kind of wildcard schema that allows everything and we lose the benefit of having content-schemas.

**Comment by boffbowsh** [comment]
@tijmenb this still needs to be a `placeholder` as it doesn't contain enough information to render the page, but other formats will need it. Placeholders will gradually disappear, and this pattern will need to happen with more and more of them as we work through migration. Creating many smaller placeholder formats will end up quite unwieldy.

I think this situation might get addressed anyway as part of the dependency resolution work - it'll be necessary to refactor some of the content schemas, maybe allowing some form of inheritance.

**Comment by boffbowsh** [comment]
Had a chat with @tijmenb offline, what we've decided is that for now we'll carry on using the placeholder format for this, but add the fields explicitly to the schema and make them not required. We can then set `additionalProperties` back to false. There's prior art for this, see the `start_date` and `end_date` in the placeholder which is used for Topical Events.

**Comment by tijmenb** [comment]
:+1:

File: formats/placeholder/frontend/examples/organisation.json (hunk 4 above)
Comment (boffbowsh) [inline formats/placeholder/frontend/examples/organisation.json]: This sounds sensible to me.

**Comment by tvararu** [comment]
I think this should be sorted out now. :raised_hands:

**Comment by boffbowsh** [comment]
This needs rebasing

**Comment by tvararu** [comment]
:green_apple:

**Pull request merged**